core/main.go

```diff
@@ -250,7 +254,11 @@ func (a *AutoSpotting) EventHandler(event *json.RawMessage) {
 			log.Println("Couldn't get instance ID of newly launched instance", err.Error())
 			return
 		} else if instanceID != nil {
-			logger.SetPrefix(fmt.Sprintf("ST:%s ", *instanceID))
+			if len(a.config.sqsReceiptHandle) == 0 {
+				logger.SetPrefix(fmt.Sprintf("ST:%s ", *instanceID))
```

Comment:
what does ST and SQ stand for in the logger prefix?

I'd rather spell entirely whatever that stands for.

Reply:
ST is there from the event branch, is standing for "start".
About SQ, as internally sqs trigger, after having "extracted" cloudwatch event from the message, is handled as a spot instance start event, i wanted a way to identify in the log that the event source is sqs.
I am using two uppercase letter to avoid a too longer prefix.
In the current `EventHandler` refactor (not yet pushed) i have changed them to:
```
EC2 Instance State-change Notification -> IS
EC2 Spot Instance Interruption Warning -> II
EC2 Instance Rebalance Recommendation -> IR
AWS API Call via CloudTrail -> CT
Scheduled Event -> SC
```
We can have more spelling prefix, but should be better to have them of the same lenght and not too long.

Reply:
Agreed, but I would prefer to have these documented somewhere or at least in code comments. Probably it wouldn't hurt to have them in-line in the logs, to avoid having to look them up